### PR TITLE
use github-script for monthly.yaml action

### PR DESCRIPTION
**Pull request by mpadge**
@assignUser Can you please have a look at this workflow, and fix anything I might have got wrong? Thanks!

### CODE REVIEW

File: .github/workflows/monthly.yaml [hunk 1]
```diff
@@ -15,13 +15,17 @@ jobs:
...
-            - copy token to `pkgcheck-action` repo secrets
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+
+            github.rest.issues.createComment({
+              owner: "ropensc-review-tools",
+              repo: "pkgcheck",
+              issue_number: 123,
+              body: |
```
Comment (assignUser) [inline .github/workflows/monthly.yaml:27]: I don't think that the preprocessor will handle the body content correctly, I'll check though.

Comment (mpadge) [reply .github/workflows/monthly.yaml:27]: Thanks!

Comment (assignUser) [reply .github/workflows/monthly.yaml:27]: https://github.com/assignUser/test-repo-a/runs/5786192057?check_suite_focus=true#step:2:33

File: .github/workflows/monthly.yaml [hunk 2]
```diff
@@ -15,13 +15,17 @@ jobs:
...
+                @mpadge It's time to renew your 'RRT_TOKEN'
+                - grant both repo and workflow access
+                - copy token to `pkgcheck-action` repo secrets
```
Comment (assignUser) [inline .github/workflows/monthly.yaml:30]:
```suggestion
              body: 
                `@mpadge It's time to renew your 'RRT_TOKEN'
                  - grant both repo and workflow access
                  - copy token to \`pkgcheck-action\` repo secrets`
```
I was correct, with a template literal it works (remember to escape backticks though)

**Review by assignUser — CHANGES_REQUESTED** [review]
You don't really need set the issue permission explicitly but it doesn't hurt either so :man_shrugging:  
:+1: For using gh script instead of a marketpalce action.
Template literal works: https://github.com/assignUser/test-repo-a/actions/runs/2076692998

**Review by assignUser — APPROVED** [review]
That should do it!

**Pull request merged**